localDate.java

```diff
@@ -0,0 +1,25 @@
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+
+public class localDate {
+
+    static LocalDate myMethod(LocalDate date1, int days, int month, int year) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-mm-dd");
+        LocalDate futureDate = LocalDate.now().plusDays(days).plusMonths(month).plusYears(year);
+        return LocalDate.now().plusDays(days).plusMonths(month).plusYears(year);
+
+
+    }
+
+public class main {

```

Comment:
```suggestion
public class Main {
```

Reply:
either leave only main method in the upper class or more Main class to separate file

Reply:
fixed in 59e51e6 'changes to task2'